src/data.js

```diff
@@ -2769,6 +2769,36 @@ module.exports = [
       'MDX',
     ],
   },
+  {
+    name: 'Gabor Gyure',
+    description:
+      'Fullstack developer with lots of love for industry and engeineering in Europe. In love with boardsports, the semantic and accessible web',
```

Comment:
It looks like you have a typo in your description if you want to fix that. Also double check some of your tags match with others.

Reply:
Thanks a lot! 😅